live/live.default.xml

```diff
@@ -285,7 +285,22 @@
 
             </extractor>
 
-            <extractor name="org.dbpedia.extraction.mappings.AbstractExtractor" status="ACTIVE">
+            <extractor name="org.dbpedia.extraction.mappings.AbstractExtractor" status="PURGE">
```

Comment:
We don't have any existing `duo:abstract` triples in our Virtuoso, so that I removed the AbstractExtractor from the live config.